changelog/12281.feature.rst

```diff
@@ -0,0 +1,8 @@
+Added support for keyword matching in marker expressions.
+
+Now tests can be selected by marker keyword arguments.
+Supported values are `int`, (unescaped) `str`, `bool` & `None`.
+
+See :ref:`marker examples <marker_keyword_expression_example>` for more information.
+
+-- by :user:`lovetheguitar`.
```

Comment:
```suggestion
-- by :user:`lovetheguitar`
```

Reply:
😆 copied that one over from one of the latest fragments added during the sprint.

Reply:
That's different with a single sentence vs. a paragraph and a and a separate byline.